### ISSUE TITLE
all the tests are shown under root suite

### ISSUE THREAD

**Issue by lmanubolu** [mmisty/cypress-allure-adapter#112]
Hi Team,

most of the test cases are added to root suite in the allure report. Could you please check this.

Thanks

**Comment by mmisty** [comment]
@lmanubolu hello
This works fine in most cases - you can see examples here https://github.com/mmisty/cypress-allure-adapter-example/tree/main/examples

And Allure reports for them: https://mmisty.github.io/cypress-allure-adapter-example/

![image](https://github.com/mmisty/cypress-allure-adapter/assets/16957275/fc209bd5-07a2-4508-aa64-299f746518a5)

To understand what's wrong I beed more details. Could you please provide
- screenshots of the report
- spec file where it doesnt work
- more detailed description

thank you

**Comment by sham-christopher** [comment]
I had the same issue when I upgraded to 0.15.1. I have now rolled it back to 0.10.4 which doesn't have this problem. 

![image](https://github.com/mmisty/cypress-allure-adapter/assets/40075792/1ecda77d-844a-468d-8f09-31df349e65eb)

**Comment by AntiMacGyver** [comment]
Hi, I have the same problem. I am sending a ZIP of an example project. Well thank you
[allure-cypress.zip](https://github.com/mmisty/cypress-allure-adapter/files/15485770/allure-cypress.zip)
![image](https://github.com/mmisty/cypress-allure-adapter/assets/170086586/0fd82945-dd1f-4365-a429-b8e76cb032bd)

**Comment by mmisty** [comment]
@AntiMacGyver hi, 
Thank you for reporting!

Since from my side everything works fine and I also posted examples (above in this thread) to understand what's happening I need more info - could you please add additional info
 - I need test (spec file) text for which you get root suite
 - package.json
 - e2e.ts/js (plugins setup)
 - support file
 
Ideally, it would be great to have repo with reproducible example

**Comment by mmisty** [comment]
@AntiMacGyver 
Ok, I see that you don't have baseUrl setup  and your tests uses different base url.
You use different urls in tests in the same file:
 - cy.visit('https://example.cypress.io')
 - cy.visit('https://testpgsk.partnersgroup.sk/prihlasenie');

If that's not critical to have them all in one file I suggest:
 - split tests this way so they have same origin in the file
 - setup baseUrl in cypress config :
 ```
e2e: {
        baseUrl: 'https://example.cypress.io', // or whatever you have
 ```

When baseUrl is not set cypress restarts browser and starts test again, and that's a problem for reporter (since previous test was not finished properly) 
If proposed solution doesn't work for you I'll think what else I can do.

**Comment by lmanubolu** [comment]
@mmisty  I set the baseURL and all the tests are from same domain but still seeing the same issue. It was working fine earlier, all of a sudden I am seeing this behavior

**Comment by mmisty** [comment]
@lmanubolu 
Could you please send a link to repository with reproducible example?

**Comment by mmisty** [comment]
Ok, i was able to reproduce that. Confirming that it is a defect. Looking.

**Comment by lmanubolu** [comment]
Thank you. will be waiting for the fix.

**Comment by mmisty** [comment]
new version with fixed is going to be released soon (0.16.5), I'll post additionaly

**Comment by mmisty** [comment]
released in 0.16.5
